frontend/packages/dev-console/src/components/formik-fields/field-types.ts

```diff
@@ -79,3 +79,8 @@ export interface SecretKeyRef {
     name: string;
   };
 }
+
+export interface RadioButtonProps extends FieldProps {
+  value: string;
+  checked?: boolean;
```

Comment:
You don't need to send `checked` as prop here as that can be derived using field.value and value.

Reply:
I've updated the `RadioButtonProps` interface. It doesn't have the `checked` prop now.